cpp/src/parquet/bloom_filter_test.cc

```diff
@@ -326,14 +327,137 @@ TEST(XxHashTest, TestBloomFilter) {
   uint8_t bytes[32] = {};
 
   for (int i = 0; i < 32; i++) {
-    ByteArray byteArray(i, bytes);
+    ByteArray byte_array(i, bytes);
     bytes[i] = i;
 
     auto hasher_seed_0 = std::make_unique<XxHasher>();
-    EXPECT_EQ(HASHES_OF_LOOPING_BYTES_WITH_SEED_0[i], hasher_seed_0->Hash(&byteArray))
+    EXPECT_EQ(HASHES_OF_LOOPING_BYTES_WITH_SEED_0[i], hasher_seed_0->Hash(&byte_array))
         << "Hash with seed 0 Error: " << i;
   }
 }
 
+// Same as TestBloomFilter but using Batch interface
+TEST(XxHashTest, TestBloomFilterHashes) {
+  uint8_t bytes[32] = {};
```

Comment:
Please use some kind of const instead of magic numbers.
```suggestion
  constexpr int kNumValues = 32;
  uint8_t bytes[kNumValues] = {};
```